requirements_test_all.txt

```diff
@@ -136,6 +136,9 @@ pymonoprice==0.3
 # homeassistant.components.binary_sensor.nx584
 pynx584==0.4
 
+# homeassistant.components.conversation
+python-Levenshtein==0.12.0
```

Comment:
Why is this added? where is it used?

Reply:
Is used in fuzzywuzzy as a faster matching method.